### ISSUE TITLE
$NVIM_QT_RUNTIME_PATH not set in Windows

### ISSUE THREAD

**Issue by xelra** [equalsraf/neovim-qt#237]
On Windows `$NVIM_QT_RUNTIME_PATH` doesn't seem to be set, which results in the neovim-gui-shim plugin not being loaded. In my case it should point to `C:\Program Files (x86)\Neovim-Qt\share\nvim-qt\runtime`.

`$VIMRUNTIME` is set and `C:\Program Files\Neovim\share\nvim\runtime` is properly loaded to the runtimepath.

**Comment by xelra** [comment]
I've experimented a bit and added `NVIM_QT_RUNTIME_PATH` to the user environment variables.

I can print from inside Neovim-Qt now:
```
:echo $NVIM_QT_RUNTIME_PATH
C:\Program Files (x86)\Neovim-Qt\share\nvim-qt\runtime
```
The path is however still not added to the runtimepath.

**Comment by xelra** [comment]
Seems like Neovim-Qt does actually try to add `C:\Program Files (x86)\Neovim-Qt\share\nvim-qt\runtime` to the runtimepath, but does so unsuccessfully. In the runtimepath there's `C:\Program`.

So this seems to be a quoting or escaping issue.

**Comment by justinmk** [comment]
This seems to be the problem: https://github.com/equalsraf/neovim-qt/blob/489416f130c8b7d9297890ddf6f0e2fff940643f/src/gui/app.cpp#L188

Another case: https://github.com/equalsraf/neovim-qt/blob/489416f130c8b7d9297890ddf6f0e2fff940643f/src/gui/app.cpp#L182

Easiest and most robust approach might be:

```c
neovimArgs.insert(1, QString("let &rtp.='%1'")
    .arg(NVIM_QT_RUNTIME_PATH));
```

**Comment by equalsraf** [comment]
Started a PR #238, @xelra can you check if this works on Windows?

**Comment by xelra** [comment]
This is fixed by #238.
